src/ray/raylet/scheduling/local_resource_manager.cc

```diff
@@ -298,9 +298,15 @@ std::optional<syncer::RaySyncMessage> LocalResourceManager::CreateSyncMessage(
   auto total = resources.total.GetResourceMap();
   resources_data.mutable_resources_total()->insert(total.begin(), total.end());
 
-  auto available = resources.available.GetResourceMap();
-  resources_data.mutable_resources_available()->insert(available.begin(),
-                                                       available.end());
+  for (const auto &[resource_name, available] : resources.available.GetResourceMap()) {
+    // Resource availability can be negative locally but treat it as 0
```

Comment:
It's the same behavior before syncer right? In terms of behavior, it will be the same as before

Reply:
Yes, same behavior before syncer.